benchmarks/automated/README.md

```diff
@@ -141,38 +138,90 @@ The final benchmark report will be available in markdown format as `report.md` i
+| processors        | Specify whether a CPU or a GPU processor should be used. Accordingly, a CPU or a GPU container will be used | "cpu" or "gpus":"<num_of_gpus_to_use>" |
+| instance_types    | Note: This is specified outside of model config. Any valid ec2 instance type may be used                    | e.g. ,"p3.8xlarge", "c4.4xlarge"       |
+
+In order to benchmark only this new model, specify the `--run-only` option.
+```
+python benchmarks/automated/run_benchmark.py --local-execution --run-only <model_name> --local-instance-type p3.8xlarge
+```
+`Note: The --run-only options runs all the models that contain the string specified as the argument`
+
 
 ## Features of the automation:
 1. To save time by *not* creating new instances for every benchmark run for local testing, use the '--do-not-terminate' flag. This will automatically create a file called 'instances.yaml' and write instance-related data into the file so that it may be re-used next time.
 ```
-python test/benchmark/run_benchmark.py --do-not-terminate
+python benchmarks/automated/run_benchmark.py --do-not-terminate
 ```
 
 2. To re-use an instance already recorded in `instances.yaml`, use the '--use-instances' flag:
 ```
-python test/benchmark/run_benchmark.py --use-instances <full_path_to>/instances.yaml --do-no-terminate
+python benchmarks/automated/run_benchmark.py --use-instances <full_path_to>/instances.yaml --do-no-terminate
 ```
 `Note: Use --do-not-termninate flag to keep re-using the instances, else, it will be terminated`.
 
 3. To run a test containing a specific string, use the `--run-only` flag. Note that the argument is 'string matched' i.e. if the test-name contains the supplied argument as a substring, the test will run. 
 ```
 # To run mnist test
-python test/benchmark/run_benchmark.py --run-only mnist
+python benchmarks/automated/run_benchmark.py --run-only mnist
 
 # To run fastrcnn test
-python test/benchmark/run_benchmark.py --run-only fastrcnn
+python benchmarks/automated/run_benchmark.py --run-only fastrcnn
 
 # To run bert_neuron and bert
-python test/benchmark/run_benchmark.py --run-only bert
+python benchmarks/automated/run_benchmark.py --run-only bert
 
 # To run vgg11 test
-python test/benchmark/run_benchmark.py --run-only vgg11
+python benchmarks/automated/run_benchmark.py --run-only vgg11
 
 # To run vgg16 test
-python test/benchmark/run_benchmark.py --run-only vgg16
+python benchmarks/automated/run_benchmark.py --run-only vgg16
 ```
 
 4. You can benchmark a specifc branch of the torchserve github repo by specifying the flag `--use-torchserve-branch` e.g., 
 ```
-python test/benchmark/run_benchmark.py --use-torchserve-branch issue_1115
+python benchmarks/automated/run_benchmark.py --use-torchserve-branch issue_1115
+```
+
+5. Once the docker image is built, you may choose to not have it re-built by passing the argument `--skip-docker-build` e.g.,
+```
```

Comment:
@nskool It seems we had to use this option as Torchserve docker files were broken.

Reply:
This has been fixed now, additional option to use `dockerhub_image`, is provided to directly pull an image from dockerhub.